bedrock/exp/templates/exp/firefox/mobile.html

```diff
@@ -0,0 +1,217 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+{% from "macros-protocol.html" import feature_card, hero, call_out_compact with context %}
+
+{% extends "exp/base/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('exp-firefox-mobile') }}
+{% endblock %}
+
+{% block canonical_urls %}
+  {# Experimental pages should set canonical to the original page URL. #}
+  <link rel="canonical" href="{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/mobile">
+{% endblock %}
+
+{% block page_title %}{{ _('Download the Firefox Browser on your Mobile for iOS and Android') }} | {{ _('Firefox') }}{% endblock %}
```

Comment:
I don't suggest changing this now as you have everything in place already, but we're on the verge of switching to a new L10n system, and these pages are never going to be translated. It doesn't hurt leaving the `_()` calls in the templates, but they are going to become redundant soon and eventually removed later down the road. We should probably try and resist adding them where they're not needed.